cmd/meroxa/root/apps/deploy.go

```diff
@@ -600,7 +606,45 @@ func (d *Deploy) prepareAppName(ctx context.Context) string {
 	return appName
 }
 
+func (d *Deploy) waitForDeployment(ctx context.Context) error {
+	startTime := time.Now().UTC()
+	logs := []string{}
+	for time.Now().UTC().Sub(startTime) < 5*time.Minute {
+		var deployment *meroxa.Deployment
+		deployment, err := d.client.GetLatestDeployment(ctx, d.appName)
+		if err != nil {
+			return err
+		}
+		newLogs := strings.Split(deployment.OutputLog.String, "\n")
```

Comment:
What is output log? I see it in meroxa-go log but the platform doesn't do anything with that. I'd rather we don't use this until it is implemented

Reply:
the worker doesn't add any outlogs right now, but in the future i would expect one log per step. we want the logs to roughly match to the CLI logs today